puppet/modules/jenkins_job_builder/files/theforeman.org/pipelines/release/releaseTarballs.groovy

```diff
@@ -0,0 +1,61 @@
+pipeline {
+    options {
```

Comment:
I think we will want an `agent` declaration always at the top of this to ensure the job runs on a slave.

Reply:
can stages still run in parallel if they're all on the same node?